src/main/java/de/medizininformatikinitiative/fhir_data_evaluator/ComponentKeyPair.java

```diff
@@ -1,29 +1,31 @@
 package de.medizininformatikinitiative.fhir_data_evaluator;
 
-import org.hl7.fhir.r4.model.CodeableConcept;
 import org.hl7.fhir.r4.model.MeasureReport;
 
 import java.util.Objects;
 
 /**
  * Represents a found value of a resource.
+ *
+ * @param definitionCode the code of the stratifier component
+ * @param valueCode the value found in the resource
  */
```

Comment:
I would call this StratumComponent because it is a component of a stratum. It also just consists of code and value like in the MeasureReport. Also Codee and Value are just hashableCodings and not ComponentKeys.